### PR TITLE
Azure - Add time and money expectation

### PR DESCRIPTION
**Pull request by jasmingoh**
This is required for #1111 (Add payment & time expectation)

**@xtinah-w's mock:**
image

**Before:**
image
image

**After:**
image
image

### CODE REVIEW

**Comment by jcoc611-microsoft** [comment]
I'm guessing there is no Azure API for getting the current values of these offerings? If not, do we need to run this through CELA for approval?

File: src/client/src/mockData/azureServiceOptions.ts [hunk 1]
```diff
@@ -160,6 +159,17 @@ export const servicesEnum = {
...
 };
 
+export const servicesExpectation = {
+  APP_SERVICE: {
+    price: "Free 30 Day Trial",
+    time: "Under 3 minute set-up time"
+  },
+  COSMOS_DB: {
+    price: "12 months of free access",
+    time: "Under 10 minutes set-up time"
```
Comment (jsondoo) [inline src/client/src/mockData/azureServiceOptions.ts]: should we localize these strings

**Comment by ali-kha** [comment]
Probably should add aria labels and alt texts to the SVGs

**Comment by jasmingoh** [comment]
@jcoc611-microsoft we will bring it up during our sponsor sync meeting tomorrow, thanks for pointing it out

File: src/client/src/types/option.d.ts [hunk 2]
```diff
@@ -17,4 +17,6 @@ export interface IOption {
   unselectable?: boolean;
   isValidTitle?: boolean;
   author?: string;
+  expectedTime?: string | FM.MessageDescriptor;
+  expectedPrice?: string | FM.MessageDescriptor;
```
Comment (streetcoder123) [inline src/client/src/types/option.d.ts:21]: What is FM.MessageDescriptor?

Comment (jasmingoh) [reply src/client/src/types/option.d.ts:21]: oh, just look at line 4!

File: src/client/src/mockData/azureServiceOptions.ts [hunk 3]
```diff
@@ -59,6 +59,14 @@ export const azureMessages = defineMessages({
     defaultMessage:
       "Connect your web app to a distributed database service to access and query data using SQL or MongoDB API."
   },
+  cosmosTimeExpectation: {
+    id: "cosmosDb.timeExpectation",
+    defaultMessage: "12 months of free access"
```
Comment (SahilTara) [inline src/client/src/mockData/azureServiceOptions.ts:64]: It's not 12 months of free access forever though right (see later sentence for explanation of what I mean)? And the free access has limits right? Ignoring everything I know about Azure, I think this message conveys that I can use as much of cosmos  possibly even 10 Gb-1 Tb of storage for free for 12 months, which I don't think is true. Also may convey that every time the user creates a new instance they get 12 month's free which I'm not sure is true either. Correct me if i'm wrong here. I think you should have some more clear wording for these things or an annotation saying that it's only free from the first instance's launch up to a certain point, also linking the hard limits per month if any.

**Comment by jasmingoh** [comment]
@jcoc611-microsoft we've just talked to our sponsors about it. App Service has been offering the 30 day free trial for as long as they remember. As for Cosmos DB, I'll work with @xtinah-w to figure out better wordings such that it is not misleading. @SahilTara

**Comment by Tanya0609** [comment]
cosmos-order

**Comment by jasmingoh** [comment]
please don't review this first - still waiting for updated design

**Pull request closed without merging**